tests/functional/test_s3.py

```diff
@@ -1340,6 +1340,79 @@ def test_500_error_with_non_xml_body(self):
             self.assertEqual(len(http_stubber.requests), 2)
 
 
+class TestS3Parser(BaseS3OperationTest):
+    def get_operations_with_expires(self):
+        s3 = self.session.create_client("s3")
+        service_model = s3.meta.service_model
+        operations_with_expires = []
+        operations = service_model.operation_names
+        for operation in operations:
+            operation_model = service_model.operation_model(operation)
+            output_shape = operation_model.output_shape
+            if output_shape and 'Expires' in output_shape.members:
+                operations_with_expires.append(operation)
+        return operations_with_expires
+
+    def test_valid_expires_value_in_response(self):
+        operations_with_expires = self.get_operations_with_expires()
+        expires_value = "Thu, 01 Jan 1970 00:00:00 GMT"
+        for operation in operations_with_expires:
+            mock_headers = {'expires': expires_value}
+            s3 = self.session.create_client("s3")
+            with ClientHTTPStubber(s3) as http_stubber:
+                http_stubber.add_response(headers=mock_headers)
+                try:
```

Comment:
What is the purpose of this try/except?  Since we are stubbing this, shouldn't it always be passing?  And if it does fail with an error, it will fail the test anyway.